matroska/19.md

```diff
@@ -6,8 +6,15 @@ ASCII-only data in string.
 
 ## Definition
 
-The string element is limited to certain byte ranges of ASCII plus a trailing optional null byte.
+The string element is limited to certain byte ranges of ASCII plus trailing optional null bytes.
 
-## Note
+## Reproducibility
+
+```sh
+./ffmpeg -f lavfi -i "color=c=black:size=2x2" -vframes 1 -pix_fmt gray8 -write_crc32 0 -c:v ffv1 -level 3 -slices 1 /tmp/EBML-ASCII-ONLY-IN-STRING-tmp.mkv
+xxd -p -c 1000000 /tmp/EBML-ASCII-ONLY-IN-STRING-tmp.mkv > /tmp/EBML-ASCII-ONLY-IN-STRING-tmp.dump
+sed -e 's/9C83756E64/9C83E282AC/g' /tmp/EBML-ASCII-ONLY-IN-STRING-tmp.dump > /tmp/EBML-ASCII-ONLY-IN-STRING-tmp2.dump
+xxd -p -c 1000000 -r /tmp/EBML-ASCII-ONLY-IN-STRING-tmp2.dump > /tmp/EBML-ASCII-ONLY-IN-STRING.mkv
```

Comment:
We use sfk for binary sed in other places. I suggest to replace these four lines of code with two.

```sh
ffmpeg -f lavfi -i "color=c=black:size=2x2" -vframes 1 -pix_fmt gray8 -write_crc32 0 -c:v ffv1 -level 3 -slices 1 EBML-ASCII-ONLY-IN-STRING.mkv
sfk replace EBML-ASCII-ONLY-IN-STRING.mkv -bin '/9C83756E64/9C83E282AC/' -yes
```